web-admin/src/features/bookmarks/Bookmarks.svelte

```diff
@@ -64,6 +65,11 @@
     metricsViewName,
   );
 
+  $: urlForExploreYAMLDefaultState = createYAMLConfigExploreUrlSearch(
```

Comment:
Nit: it'd be nice if the function name more closely matched the name of the resultant object